src/routes/blog/post/personal-chatbot-gpt-4o/+page.markdoc

```diff
@@ -0,0 +1,275 @@
+---
+layout: post
+title: Build a personal chatbot with GPT-4o and Appwrite Functions
+description: Learn how to use OpenAI's new flagship model, GPT-4o to create an intelligent chatbot.
+date: 2024-05-17
+cover: /images/blog/personal-chatbot-gpt-4o/cover.png
+timeToRead: 5
+author: aditya-oberai
+category: functions
+---
+
+Recently, at the OpenAI Spring Update, OpenAI CTO Mira Murati announced the launch of their new flagship model, **GPT-4o**. GPT-4o happens to be OpenAI's fastest and most affordable model so far, which led us to wonder if we could use it to build our very own chatbot. 
+
+Therefore, in this blog, we will learn how to use Appwrite Functions and the OpenAI GPT-4o API to build a personal chatbot.
+
+## Setting up the OpenAI Platform
+
+To use the OpenAI GPT-4o API, we must first create an API key on their platform. To do that, we must first create an account on the [OpenAI platform](https://platform.openai.com/). Once the account is set up and a project is created, we can visit their [API keys](https://platform.openai.com/account/api-keys) page and create an API key. Ensure you copy and save this key in a safe place, as the OpenAI platform will not let you view the key after it is created.
+
+![Create an OpenAI API Key](/images/blog/personal-chatbot-gpt-4o/openai.png)
+
+> Note: To use the GPT-4o API, your account must be upgraded to the Usage tier 1 (you must purchase at least $5 worth of credits). To learn more, visit their [Usage tiers documentation](https://platform.openai.com/docs/guides/rate-limits/usage-tiers?context=tier-one).
```

Comment:
"the Usage tier 1" sounds a bit odd to me. Maybe:

```suggestion
> Note: To use the GPT-4o API, your account must be upgraded to Usage tier 1 (you must purchase at least $5 worth of credits). To learn more, visit their [Usage tiers documentation](https://platform.openai.com/docs/guides/rate-limits/usage-tiers?context=tier-one).
```

or

```suggestion
> Note: To use the GPT-4o API, your account must be upgraded to the Tier 1 Usage tier (you must purchase at least $5 worth of credits). To learn more, visit their [Usage tiers documentation](https://platform.openai.com/docs/guides/rate-limits/usage-tiers?context=tier-one).
```

Reply:
@stnguyen90 `Usage tier 1` is language taken from their documentation